.github/workflows/devcontainer-ci.yml

```diff
@@ -16,8 +16,42 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v4
       with:
-        ref: ${{ github.event.pull_request.merge_commit_sha }}
-      
+        fetch-depth: 0  # Fetch full history for merge testing
+        
+    - name: Create fresh merge commit for testing
```

Comment:
GitHub is giving us this in merge queues. What's the reason for doing it on our own?